### PR TITLE
Use openshift v1 api as default

### PR DESCRIPTION
**Pull request by oschreib**
Move to openshift v1 api and bump version to 0.0.7

### CODE REVIEW

**Comment by oschreib** [comment]
cc @simon3z

**Comment by simon3z** [comment]
@oschreib can you address travis failures? I suppose they're related to the url entrypoint change.

File: test/test_project.rb [hunk 1]
```diff
@@ -11,14 +11,14 @@ def test_get_project
       .to_return(body: open_test_json_file('project_b1.json'),
                  status: 200)
 
-    client = OpenshiftClient::Client.new 'https://localhost:8080/osapi'
-    project = client.get_project 'master'
+    client = OpenshiftClient::Client.new 'https://localhost:8080/oapi' 'v1'
```
Comment (abonas) [inline test/test_project.rb]: why did you add the version?

**Comment by abonas** [comment]
@oschreib , except for minor comment - ack.
